event-handlers/src/main/java/no/sikt/nva/nvi/events/cristin/ScientificPerson.java

```diff
@@ -17,6 +17,12 @@ public final class ScientificPerson {
     private String subDepartmentIdentifier;
     @JsonProperty("gruppenr")
     private String groupIdentifier;
+    @JsonProperty("forfattervekt")
+    private String authorPoints;
```

Comment:
`authorAffiliationPoints` or `pointsForAffiliation`?

Reply:
In cristin it is points per author at affiliations. I feel it is wrong to name pointsForAffiliation since we will probably have multiple pointsForAffilition per candidate.